docs/DeveloperGuide.md

```diff
@@ -740,58 +739,109 @@ testers are expected to do more *exploratory* testing.
 ### Launch and shutdown
 
 1. Initial launch
-
    1. Download the jar file and copy into an empty folder
+   2. Open up a terminal, and run the following command: `java -jar DocTrack.jar`
+      - Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
-
-1. Saving window preferences
-
+2. Saving window preferences
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
-
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
 ### Adding a person
 
+1. Adding a person
+   1. Prerequisites: None
+   2. Test case:`add person n/Elmo p/98765432 e/elmo@sesa.me a/Sesame Street st/recovering t/insurance`
+   3. Expected: A new contact is added to the list.
+   - The status bar shows the following: "New person added: Elmo; Phone: 98765432; 
```

Comment:
Should we explain what `status bar` is, so that it is clearer for users? And should we distinguish it from `command box`?

Reply:
This is for developers though, so would it not be clear enough?

Additionally, do you think it would be good to have a glossary then, to show what name we used to refer to the UI sections?

Reply:
Yeah perhaps a glossary works, or we can use `ResultDisplay`

Reply:
> Should we add another section to explain the parts of the GUI?

We can do this.